Composer/packages/server/src/models/asset/__tests__/assetManager.test.ts

```diff
@@ -140,6 +142,7 @@ describe('assetManager', () => {
             packageName: 'generator-conversational-core',
             packageSource: 'npm',
             packageVersion: '1.0.3',
+            availableVersions: [''],
```

Comment:
Since we are adding the functionality to process multiple versions of packages. Would it better to write more tests asserting around multiple package versions?

Reply:
Good point. I have added a unit test for the `getNpmPackageVersions` function. I think that should suffice seeing as this is a function and feature that will only be available to internal devs who set the hidden feature flag. If we make this feature GA then at that point we can write a more robust test suite that tests version selection in the UI all the way through creation execution.